util: Coverage Report additions

valarnin opened the pull request:
- Add "other" content to coverage report
- Add spacer row between expansions/other triggers
- Have coverage report use zone label where appropriate
- Clean up duplicate `kPrefixToCategory`
- Add `Comments` column and associated field to `TriggerSet`
- Add `Status` column

`Status` isn't really a good name for the column but it's 4:30am and I'm out of brainpower to think about things like names.

Hovering over the `Status` column's values will show the date/time of the commit (for unreleased) or tag (for released).

Example comment in screenshot was not actually committed.

![image](https://github.com/user-attachments/assets/49907e04-2545-496c-ad54-380ef2db9a4c)

Akurosia commented:
Would "Release Version" a better name instead of Status?
Also for tags, would there be a benefit of maybe adding a link to the github page?

valarnin commented:
Renamed column, added colors, added PR info for open PRs. Maybe it makes more sense to list the open PRs first and the date second?

From an information standpoint I don't think there's any more I can do here.

Linking to a give PR/release/etc doesn't really work very well with the current table's structure, I think we'd need to rewrite the entire presentation layer to have that make sense. (css grid is making this frustrating to deal with)

![image](https://github.com/user-attachments/assets/d0f8db8d-1b6a-48d6-8fb7-be0835a865b4)

JLGarber commented:
Probably worth reversing open PRs and the date, yes, but not a big deal either way. I think making it clear what is and isn't released yet is the big win here.
